### PR TITLE
Add Highbury_710

### PR DESCRIPTION
**Pull request by sudopunk**
Ref: https://cosmos.directory/highbury

### CODE REVIEW

**Comment by HeesungB** [comment]
@sudopunk Hello. 
Can you edit the file using the link below?
https://github.com/chainapsis/keplr-chain-registry#requirement-details

You can also use the command below to check the validation locally.
```
yarn validate cosmos/highbury_710.json
```

**Comment by sudopunk** [comment]
> @sudopunk Hello. Can you edit the file using the link below? https://github.com/chainapsis/keplr-chain-registry#requirement-details
> 
> You can also use the command below to check the validation locally.
> 
> ```
> yarn validate cosmos/highbury_710.json
> ```

I have updated the config based on the requirements.
The command executed with no messages or errors.

File: cosmos/highbury_710.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
...
+  "chainId": "highbury_710-1",
+  "chainName": "Highbury",
+  "chainSymbolImageUrl": "https://raw.githubusercontent.com/chainapsis/keplr-chain-registry/main/images/highbury_710/chain.png",
+  "nodeProvider": {
+      "name": "Blackfury Genesis Validator",
+      "email": "blackfuryan@skiff.com",
+      "website":"https://fury.black/"
+    },
+  "walletUrl": "https://wallet.keplr.app/chains/highbury",
+  "walletUrlForStaking": "https://wallet.keplr.app/chains/highbury",
```
Comment (HeesungB) [inline cosmos/highbury_710.json:13]: @sudopunk Could you remove this two(`walletUrl`, `walletUrlForStaking `) filed?
This is an invalid URL and is not needed now.

Comment (sudopunk) [reply cosmos/highbury_710.json]: Yes, removed.

**Pull request merged**